client/game.hpp

```diff
@@ -44,6 +44,18 @@ class Game {
 
 	// Frames per second
 	unsigned int fps;
+	float avg_frame_time;
+
+	// Elapsed time spend on each section
+	float outside_game_time 	= 0.0;
```

Comment:
It would be create to create a data structure of some kind to hold this data. You could then store a queue of these to create a graph in the UI.